cpp/ps0002/problem solve 2.cpp

```diff
@@ -1,15 +1,17 @@
 #include <iostream>
 using namespace std;
 int main(){
-    int people=18;
-    int medicine=7;
-    int team=10;
-    float baked=8.5;
-    char name='H';
-    double born=9020000;
-    double divition=2.428;
-    float bill=999.75;
-    double age=159.068;
-  return 0;
-
-}
\ No newline at end of file
+    short people=18; // 0.5/0.75
+    short medicine=7; // 0.75/0.75
+    short team=10; // 0.75/0.75
+    float baked=8.5; // 0.75/0.75
+    char name='H'; // 0.75/0.75
+    int born=9020000; // 0.75/0.75
+    double division=2.428; // 0.75/0.75
+    float bill=999.75; // 0.75/0.75
+    unsigned int age=15945068; // 0.5/0.75
+    // total 6.25/6.75
+    
+    return 0;
+}
+// great work 😎 8.5/10 ⭐

```

Comment:
- [x] program is in the correct path
- [x] no syntax errors
- [x] formatted code to be clear and readable
- [ ] commented some phrases for blocks of code